docs/SONATA_DEVELOPER_GUIDE.md

```diff
@@ -1238,9 +1233,7 @@ node_id mapping is specified according to [description below](#gid_mapping).
             "model_type": "point",
             "node_id": [1, 2, 3, 5, 7, 9, ...]
         }
-        "layer4": {
-            "gids": [1, 2, 3, 4, 5, ...]
-        },
+        "combined": ["bio_layer45", "V1_point_prime"]
```

Comment:
Is "combined" a new node-set?

Reply:
It's just a name for an example of a compound node set, as there was example of this before. I couldn't think of a better name for it.